bbc/simorgh
# WSTEAM1-366 - Add `noindex` meta tag to App version

Resolves https://jira.dev.bbc.co.uk/browse/WSTEAM1-366

Overall changes
======
- Adds `noindex` meta tag on `.app` pages to prevent search engine crawling

Code changes
======
- Updates the server rendered `Document` components to include the new `<meta />` tag to prevent search engine crawling on `.app` versions of pages

Testing
======
1. Visit http://localhost:7080/kyrgyz/articles/c4n81n8rxyeo.app?renderer_env=live
2. Inspect the HTML using `View page source` or via the Chrome dev tools
3. Confirm that the `<meta name="robots" content="noindex" />` tag renders in the `<head>` element

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
